docs/release-notes.rst

```diff
@@ -6,6 +6,7 @@ Release notes
 
 Release 0.10.1 (unreleased)
```

Comment:
Need to remember and bump to 0.11.0 since we are changing the interface of `BatchedDataLoader` (removing arguments). No action for this diff.

Reply:
sure thing.